src/net/sourceforge/kolmafia/session/EventManager.java

```diff
@@ -100,27 +91,26 @@ public static boolean addNormalEvent(String eventHTML, boolean addTimestamp) {
     // href='showplayer.php?who=115875'><b><font color=green>Brianna</font></b></a> has played a
     // song (The Polka of Plenty) for you.
 
-    // Remove tags that are not hyperlinks
+    var eventText =
+        eventHtml
+            // Add a space before item acquisition
+            .replace("<center><table class=\"item\"", " <table")
+            // Remove tags that are not hyperlinks
+            .replaceAll("</?[^aA/][^>]*>", "")
+            // Replace links to profiles with a player descriptor
+            .replaceAll("<a[^>]*showplayer\\.php\\?who=(\\d+)[^>]*>(.*?)</a>", "$2 (#$1)")
+            // Remove the rest of the tags
+            .replaceAll("<[^>]*>", "");
 
-    eventHTML = eventHTML.replaceAll("</[^aA][^>]*>", "");
-    eventHTML = eventHTML.replaceAll("<[^aA/][^>]*>", "");
-
-    String eventText =
-        eventHTML.replaceAll("<a[^>]*showplayer\\.php\\?who=(\\d+)[^>]*>(.*?)</a>", "$2 (#$1)");
-
-    eventText = eventText.replaceAll("<.*?>", "");
-
-    if (addTimestamp) {
-      EventManager.eventTexts.add(
-          EventManager.EVENT_TIMESTAMP.format(new Date()) + " - " + eventText);
-    } else {
-      EventManager.eventTexts.add(eventText);
-    }
+    EventManager.eventTexts.add(eventText);
```

Comment:
Before the refactor, this seems to have checked `addTimestamp` and prepended the date if set.

Reply:
Ah yeah, I had kept this behaviour and then slowly removed it as I tried to fix other unexpected changes that causes test failures. Fixed now